.github/workflows/release-on-tag.yml

```diff
@@ -199,6 +199,6 @@ jobs:
       # No idea why this happens, but let's just keep a single cargo release
       # to avoid this issue and to simplify the workflow, because the problem
       # we tried to protect against is not that likely to happen anyway.
-      - run: cargo release publish --no-confirm --no-verify --no-push --no-tag --allow-branch '*'
+      - run: cargo release publish --no-confirm --no-verify --allow-branch '*'
```

Comment:
I think this version requires the `--execute` flag, to actually execute the publish command. At least it required that, when I ran it locally

Reply:
Right, I'll fix it in a separate PR

Reply:
Created https://github.com/rust-marker/marker/pull/323